modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
```

Comment:
Please check that this file belongs in the PR.  I do not see it listed as an include in the assembly that is part of the PR.